client/src/components/categorical/categorical.js

```diff
@@ -5,18 +5,27 @@ import { connect } from "react-redux";
 
 import Category from "./category";
 
-@connect(state => {
-  const ranges = _.get(state.controls.world, "summary.obs", null);
-  return {
-    ranges
-  };
-})
-class Categories extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
+/* Cap the max number of displayed categories */
+const truncateCategories = options => {
+  const maxOptionsToDisplay = 100; // could be a property if helpful....
```

Comment:
prefer this as a constant in globals, several others of the same kind are there

Reply:
I put it in the component as it is very specific to the component (and not a "global" constant).    Globals should be, you know, global :-)

That said, I don't feel strongly and will make the change.